scripts/generate-date-info.js

```diff
@@ -0,0 +1,176 @@
+import fs from "fs";
+
+const output = {
+    _: {
+        y: "Y",
+        m: "M",
+        d: "D",
+        o: "ymd",
+        s: ["-", "-"],
+        w: 0,
+    },
+};
+
+const firstDays = JSON.parse(
+    fs.readFileSync("node_modules/cldr-core/supplemental/weekData.json"),
+).supplemental.weekData.firstDay;
+const dayNums = {
+    sun: 0,
+    mon: 1,
+    tue: 2,
+    wed: 3,
+    thu: 4,
+    fri: 5,
+    sat: 6,
+};
+
+const modernLocales = JSON.parse(
+    fs.readFileSync("node_modules/cldr-core/availableLocales.json"),
```

Comment:
Can we just do a json top level import?

Reply:
Good idea, that's more clean for sure. Added.